USER_MANUAL.md

```diff
@@ -293,12 +293,12 @@ hardware, what you have tried, and the exact nature of the problem.
 
 You must have a modern CPU with AVX support to run FreeDV 2020.  If
```

Comment:
Hmm, now that I'm looking at this again, perhaps we should update this paragraph to indicate that the M1 Macs can also run 2020?

Reply:
Updated this section. Feel free to change if needed.